package.json

```diff
@@ -56,7 +56,7 @@
 		"check-engines": "check-node-version --package",
 		"create-symlinks": "node ./scripts/create-symlinks.js",
 		"lerna-bootstrap": "lerna bootstrap --hoist",
-		"npmpackagejsonlint": "npmPkgJsonLint ./packages -c .npmpackagejsonlintrc.json",
+		"npmpackagejsonlint": "npmPkgJsonLint ./packages",
```

Comment:
Should it be named `lint-packages`?

Reply:
What about `lint:pj`, not that it is immediately obvious what actually infers I saw it the other day alongside `lint:js`, `lint:md` etc?

```json
		"lint:js": "eslint .",
		"lint:css": "stylelint .",
		"lint:md": "remark-lint .",
		"lint:pj": "npmPkgJsonLint ./packages",
		"lint:scss": "stylelint --syntax=scss .",
```

Reply:
Or `lint:package` maybe?, could be used with `npm-run-all` `npm run all lint:*`

Reply:
`lint:pkg-json` should do the trick, and agreed on `npm-run-all`.